docs/src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
There's no way to just pull this in from npm or something instead of duplicating?

Reply:
@mvines nope, have to add it to the repo

Reply:
sad